Implementation of outgoing mirroring (#34353)
demisto/content: Packs/VaronisSaaS/Integrations/VaronisSaaS/README.md

```!varonis-close-alert  alert_id=C98A3E72-99E9-4E5C-A560-7D04FA60686E close_reason="Inaccurate alert logic"  note="Alert is irrelevant. Closed" ```

### get-mapping-fields
***
Returns the list of fields to map in outgoing mirroring. This command is only used for debugging purposes.

#### Base Command

`get-mapping-fields`
#### Input

There are no input arguments for this command.

#### Context Output

There is no context output for this command.

## Incident Mirroring

You can enable outgoing incident mirroring between Cortex XSOAR incidents and Varonis alerts (available from Cortex XSOAR version 6.0.0).
To set up the mirroring:
1. Enable *Fetching incidents* in your instance configuration.
2. In the *Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored (currently only outgoing mirroring is available):

    | **Option** | **Description** |
    | --- | --- |
    | None | Turns off incident mirroring. |

>     | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in Vsronis SaaS service (outgoing mirrored fields). |

Reviewer: ```suggestion
    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in Varonis SaaS service (outgoing mirrored fields). |
```

Newly fetched incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.

### Mirroring Out Notes
The supported fields in the mirroring out process are:
- Varonis Alert Status.
- Varonis Close Reason
- Incident Close Notes

**Important Note:**
You have two options how to close Varonis Alert:

>  - The first option is to change a Varonis Alert Status. In this case, the status of an alert will be change by a mirroring functionality, but the Incident in XSOAR won't be closed.

Reviewer: ```suggestion
 - The first option is to change the Varonis Alert Status field in the XSOAR incident. In this case, the status of the alert in Varonis SaaS service will be change by the mirroring functionality, but the Incident in XSOAR won't be closed.
```

>  - The second one is to close an Incident. In this case, the Varonis Alert will be closed on the Varonis side by the post-processing script.

Reviewer: ```suggestion
 - The second one is to close the incident in XSOAR. In this case, the Varonis Alert will be closed on the Varonis side by the post-processing script.
```
